src/content/docs/infrastructure/host-integrations/host-integrations-list/lighttpd-integration.mdx

```diff
@@ -112,72 +109,81 @@ integrations:
     # interval: 30s
     config:
       name: lighttpdFlex
-      global:
-        base_url: http://lighttpd.net/
       apis:
-        - event_type: status
-          url: status?format=plain
-        - event_type: {'<THE_PLACE_WHERE_METRICS_ARE_CAPTURED>'}
-          commands:
-            - cache: status?format=plain
-              split_by: ": "
-          snake_to_camel: true
+        - event_type: INSERT_EVENT_TYPE
+          url: INSERT_URL
 
-
-  - name: nri-flex
-    interval: 10s
-    # timeout: 5s
-    config:
-      name: test
-      apis:
-        - event_type: {'<THE_PLACE_WHERE_METRICS_ARE_CAPTURED>'}
-          commands:
-            - run: curl 'http://lighttpd.net/status?auto'
-              split_by: ": "
-              regex_matches:
-                - expression: (\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)
 ```
 
 </Step>
 
 <Step>
 
-### Restart the New Relic infra agent and lighttpd service
+### Forward Lighttpd logs to New Relic
 
-To restart the New Relic infra agent follow [these](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) instructions and run this command.
+You can use our [log forwarding](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/) to forward Lighttpd logs to New Relic.
+
+On Linux machines, your log file named `logging.yml` should be present in the path below. If you don't see your log file, then you will need to create it by following the above log forwarding documentation.
 
 ```
-sudo systemctl restart lighttpd.service && sudo systemctl restart newrelic-infra.service
+/etc/newrelic-infra/logging.d/
+```
+
+After creating the log file, add the this script to the `logging.yml` file:
```

Comment:
After creating the log file, add the below script to the `logging.yml` file:

Reply:
Hi @rhetoric101 , 
Thank you for the review.
kindly confirm if my above text suggestion is good to go.
Thank you 😃